pkg/registries/adapters/adapter.go

```diff
@@ -102,6 +102,8 @@ func imageToSpec(log *logging.Logger, req *http.Request) (*apb.Spec, error) {
 		return nil, err
 	}
 
+	spec.Image = spec.Image + ":" + apbtag
```

Comment:
I know we probably do the concat somewhere but most of the time I see the Sprintf format:

```golang
spec.Image = fmt.Sprintf("%s:%s", spec.Image, apbtag)
```